polyfill/lib/ecmascript.mjs

```diff
@@ -427,8 +429,7 @@ export const ES = ObjectAssign(ObjectAssign({}, ES2019), {
   GetEpochFromParts: (year, month, day, hour, minute, second, millisecond, microsecond, nanosecond) => {
     let ms = Date.UTC(year, month - 1, day, hour, minute, second, millisecond);
     // Date.UTC interprets one and two-digit years as being in the 20th century
-    if (year >= 0 && year < 100)
-      ms = new Date(ms).setUTCFullYear(year);
+    if (year >= 0 && year < 100) ms = new Date(ms).setUTCFullYear(year);
```

Comment:
I'm not sure I like this change, but please land either way.

Reply:
I'm personally not a fan of this one-line style either but I found it to be predominant in the existing code ... I'll land it, but we can certainly change the config file to mandate a different style and then have eslint auto-fix it.